pigeon/pigeon/settings.py

```diff
@@ -37,8 +38,7 @@
 # See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
-
+DEBUG = True
```

Comment:
In all case DEBUG should be explicitely set in local_settings. I've changed it to True in common settings because I think that the "raw" source code (just after a git clone) should reflect a dev environment and not a production one

Reply:
I agree more with @xouillet on this one after thinking. You setup your prod env once (hopefully) but dev env are meant to be spawn everytime everywhere...